specs/protocol/mpt-migration/exec-engine.md

```diff
@@ -0,0 +1,75 @@
+# L2 Execution Engine
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+- [Timestamp Activation](#timestamp-activation)
+- [EVM Changes](#evm-changes)
+  - [Transition from ZK Trie to MPT](#transition-from-zk-trie-to-mpt)
+  - [`SELFDESTRUCT` opcode](#selfdestruct-opcode)
+  - [`isSystemTransaction` boolean at transaction struct](#issystemtransaction-boolean-at-transaction-struct)
+- [Fee Distribution Process](#fee-distribution-process)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+[g-zktrie]: ../../glossary.md#zk-trie
+[g-mpt]: ../../glossary.md#merkle-patricia-trie
+[validator-system]: ../validator-v2/overview.md
+
+## Overview
+
+This upgrade changes the underlying trie structure of the execution client from the [ZK Trie][g-zktrie] to the
+[Merkle Patricia Trie][g-mpt]. There are also subsequent changes to the EVM to enhance the compatibility with
+Vanilla OP Stack.
+
+## Timestamp Activation
+
+Kroma MPT Migration upgrade, like other network upgrades, is activated at a timestamp. Changes to the L2 Block execution
+rules are applied when the `L2 Timestamp >= activation time`.
+
```

Comment:
I think the MPT migration process should be documented, could you add it as TBD for now?

Reply:
oh, i see